test/test-mdl.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>Fragment test page</title>
+    <meta mdl-fragment headers-cache-control="must-revalidate" method="GET" mode="cors">
```

Comment:
Let's pass two params to headers
```headers-cache-control="must-revalidate"``` and
```headers-custom-header="custom-value"``` and see if they fall into attributes or not.